packages/cmd/src/profiles/CliProfileManager.ts

```diff
@@ -353,7 +354,7 @@ export class CliProfileManager extends BasicProfileManager<ICommandProfileTypeCo
             this.log.debug("Setting profile field %s from command line option %s", propNamePath, optionName);
             if (secureOp && prop.secure) {
                 this.log.debug("Performing secure operation on property %s", propNamePath);
-                return secureOp(propNamePath, propValue);
+                return secureOp(propNamePath, propValue, !prop.optionDefinition.required);
```

Comment:
Should I add a test around the addition on this line? Codecov says it's already covered, but I'm not sure how.

Reply:
Sorry for not saying anything earlier. 
the line coverage won't be affected, but the branch coverage is. No real concern with adding tests now or later :)